### PR TITLE
EC 60 PCB

### PR DESCRIPTION
**Pull request by Cipulot**
Addition of the EC 60 PCB firmware. Open source universal 60% EC PCB with GH60 outline.
## Description

The EC 60 is a GH60 compatible PCB that boosts multiple layout support to be used in a variety of languages.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cipulot/ec_60/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# EC 60
+
+![EC 60 PCB](https://i.imgur.com/XNwsVOn.png)
```
Comment (keyboard-magpie) [inline keyboards/cipulot/ec_60/readme.md:3]:
```suggestion
![EC 60 PCB](https://i.imgur.com/XNwsVOnh.png)
```
Just for consistency.

**Review by keyboard-magpie — CHANGES_REQUESTED** [review]

File: keyboards/cipulot/ec_60/readme.md (hunk 1 above)
Comment (Cipulot) [reply keyboards/cipulot/ec_60/readme.md]: Done 👍

**Review by keyboard-magpie — APPROVED** [review]

File: keyboards/cipulot/ec_60/rules.mk [hunk 2]
```diff
@@ -0,0 +1,2 @@
+CUSTOM_MATRIX = lite
+SRC += analog.c matrix.c ec_switch_matrix.c
```
Comment (drashna) [inline keyboards/cipulot/ec_60/rules.mk:2]:
```suggestion
QUANTUM_LIB_SRC += analog.c 
SRC += matrix.c ec_switch_matrix.c
```

Comment (Cipulot) [reply keyboards/cipulot/ec_60/rules.mk]: Done 👍

**Comment by Cipulot** [comment]
@drashna friendly ping for review of this and the other EC PRs open 😃

**Comment by Cipulot** [comment]
@drashna sorry for the new ping but the boards are in production and soon I'll have to confirm the FW. If possible we'd like to have this and the other EC merged since the latest changes are all verified, consistent across the other PRs and working.

**Comment by keyboard-magpie** [comment]
The last merges for cipulot's boards are all hardware verified, merged under the new rules (1 reviewer approval with all conditions met).

**Pull request merged**